src/extensions/rolesHelper.ts

```diff
@@ -47,6 +50,7 @@ export function grantRoles(
           return member === `serviceAccount:${serviceAccountEmail}`;
         });
       });
+      policy.version = 3;
```

Comment:
Can `getIamPolicy` return a `.version` on its own? What would overwriting it here do?

Reply:
if it's a closed loop get-write-set, then the get with v3 will return `version: 3`.

outside mods that are a "blind set" can be done without a `version: 3`

I verified this with the IAM  person (Ajay).

So I think we can remove `version: 3` being forced here.